recipes/google-cloud-bigquery-storage/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "google-cloud-bigquery-storage" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "b0972b77ce24533758ec4a75c0d93072201fc3ccfb40519e98dbcac20150e08c" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+
+  run:
+    - python
+    - enum34;   # [ py < 34 ]
```

I think this semicolon shouldn't be here:
```suggestion
    - enum34  # [ py < 34 ]
```

---

thank you for your review !